notebooks/2022-04-03potential_bot.ipynb

```diff
@@ -0,0 +1,162 @@
+    "    port=\"3306\",\n",
+    "    database=\"serlo\"\n",
+    ")\n",
+    "\n",
+    "def flagged_word(word):\n",
+    "    flaggedWords = [\n",
+    "                    'online','official','call','fantasy','contact','installation', \n",
+    "                    'pay', 'manufacturer', 'delivery', 'established', 'registration',\n",
+    "                    'seo', 'service', 'customers', 'trading', 'company', 'buy', 'sell', \n",
+    "                    'escort', 'provider', 'product', 'patients', 'website', 'sites', \n",
+    "                    'free', 'affordable', 'website', 'buisness', 'installed', 'install',\n",
+    "                    'management', 'manufacture', 'install', 'download', 'provides',\n",
+    "                    'downloaded', 'services', 'development', 'drug', 'top', 'reviews',\n",
+    "                    'review', 'treatment', 'deliver', 'hire', 'hiring', 'supply', 'supplier', \n",
+    "                    'commercial', 'discount', 'account', 'websites', 'app', 'products', \n",
+    "                    'guests', 'deals'\n",
+    "                    ]\n",
+    "    count = len(flaggedWords)\n",
+    "    counter = 0\n",
+    "    while counter <= count:\n",
+    "        if word.lower() == flaggedWords[counter]:\n",
+    "            return True\n",
+    "            break\n",
+    "        else: \n",
+    "            counter = counter + 1\n",
+    "        if counter == count:\n",
+    "            return False\n",
+    "\n",
+    "\n",
+    "def unhelpful_word(word):\n",
+    "    unhelpfulWords = [\n",
+    "                      'col','plugin','rows','state','children','content','text',\n",
+    "                      'type','level','children','p','h','ttachment','ile','layout',\n",
+    "                      'child','image','src','width','alt','href'\n",
+    "                     ]\n",
+    " \n",
+    "    count = len(unhelpfulWords)\n",
+    "    counter = 0\n",
+    "    while counter <= count:\n",
+    "        if word.lower() == unhelpfulWords[counter]:\n",
+    "            return True\n",
+    "            break\n",
+    "        else: \n",
+    "            counter = counter + 1\n",
+    "        if counter == count:\n",
+    "            return False\n",
+    "        \n",
+    "        \n",
+    "def readable(splitString):\n",
+    "    resultData = ''\n",
+    "    for word in splitString:\n",
+    "        if word.isalpha():\n",
+    "            if unhelpful_word(word):\n",
+    "                pass\n",
+    "            else:\n",
+    "                resultData = resultData + ' ' + word\n",
+    "    return(resultData)\n",
+    "\n",
+    "\n",
+    "def description_length(description):\n",
+    "    descriptionLen = len(description)\n",
+    "    return (descriptionLen)\n",
+    "\n",
+    "def query(sql):\n",
+    "    c = db.cursor()\n",
+    "    c.execute(sql)\n",
+    "    return c.fetchall()\n",
+    "\n",
+    "potentialBots = np.empty((0,2))\n",
```

Comment:
Same suggestion about this and other vars: instead of using camel case it is more common to use snake case in Python.